### DISCUSSION TITLE
2 active licenses on 1 machine

### DISCUSSION THREAD

**Discussion by simoa97** [yuezk/GlobalProtect-openconnect#409] [General]
Hi, I encountered following problem where on my machine (Linux Mint 21), client activated 2 licenses.

I tried reinstalling and rebooting the machine, but I still cannot make the client detect the first license that was activated. This happened to me before, and after uninstalling, deleting residual directories, rebooting the machine and installing the client again, the client detected the original license I activated, but now it doesn't help. Is there any way I can fix this? Thank you,

**Comment by yuezk** [comment]
Hi @simoa97 you can manage the activations on the LemonSqueezy [My Orders](https://app.lemonsqueezy.com/my-orders) page. You can deactivate the unused one.
